crates/pop-cli/src/commands/build/mod.rs

```diff
@@ -44,15 +44,6 @@ pub(crate) struct BuildArgs {
 /// Build a parachain, smart contract or Rust package.
```

Comment:
Update docs

Reply:
Cannot point to the `BuildsArgs` struct, but do we still need the `id` field there as it is not needed for building a chain / contract and it is included in the `BuildSpecCmd`?

Reply:
Good point, `id` here is not needed anymore. Removed and update docs in https://github.com/r0gue-io/pop-cli/pull/394/commits/b0b5946973c0f24ff07ef0d539ecd381b33f8fc4